public/server/db.json

```
@@ -72,7 +72,8 @@
     },
     
     "lol":{
-        "definition": "Laugh Out Loud"
+        "definition": "Laugh Out Loud",
+        "alternatives": "Lots Of Love, Lots Of Likes, Lots Of Loosers, Lots Of Light, Lots Of Laugh
```

There seems to be an error on this line. 
```suggestion
        "alternatives": "Lots Of Love, Lots Of Likes, Lots Of Loosers, Lots Of Light, Lots Of Laugh"
```